platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -399,7 +399,7 @@ class MeasurementService {
    * @param {MeasurementSource} source The measurement source instance
    * @param {string} definition The source definition
    * @param {Measurement} measurement The source measurement
-   * @return {string} A measurement id
+   * @return void
```

Comment:
I think you switched the wrong jsdoc? This _does_ return things. The other one doesn't

Reply:
Updated, thanks.